diff --git a/node_modules/@types/rdfjs__environment/lib/Formats.d.ts b/node_modules/@types/rdfjs__environment/lib/Formats.d.ts
new file mode 100644
index 0000000..9eb8604
--- /dev/null
+++ b/node_modules/@types/rdfjs__environment/lib/Formats.d.ts
@@ -0,0 +1,12 @@
+import { DataFactory, Stream } from '@rdfjs/types';
+import { SinkMap } from '@rdfjs/sink-map';
+import { EventEmitter } from 'events';
+
+declare class Formats {
+    constructor(arg: { factory: DataFactory });
+    parsers: SinkMap<EventEmitter, Stream>;
+    serializers: SinkMap<Stream, EventEmitter>;
+    import(other: Formats): Formats;
+}
+
+export default Formats;
